Solution1/lect14.task1/StringExtension.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace lect14.task1
+{
+    static class StringExtension
+    {
+        public static void FormatString(this ClassString classString, string str)
```

Comment:
replace user defined 'ClassString' to 'string'

public static void FormatString(this string str)